.github/ISSUE_TEMPLATE/BugReport.yml

```
@@ -0,0 +1,107 @@
+name: Bug Report
+description: Is the SDK not working as expected? Help us improve by submitting a bug report.
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Ensure you go through our [troubleshooting](https://docs.datadoghq.com/real_user_monitoring/mobile_and_tv_monitoring/troubleshooting/#react-native) page before creating a new issue. 
+        Before getting started, if the problem is urgent or easier to investigate with access to your organization's data please use our [official support channel](https://www.datadoghq.com/support/).
+  - type: textarea
+    id: description
+    attributes:
+      label: Describe the bug
+      description: Provide a clear and concise description of what the bug is.
+    validations:
+      required: true
+  - type: textarea
+    id: reproduction
+    attributes:
+      label: Reproduction steps
+      description: |
+        Provide a self-contained piece of code demonstrating the bug. 
+        For a more complex setup consider creating a small app that showcases the problem. 
+        **Note** - Avoid sharing any business logic, credentials or tokens.
+    validations:
+      required: true
+  - type: textarea
+    id: logs
+    attributes:
+      label: SDK logs
+      description: |
+        Please provide console logs before, during and after the bug occurs. 
+    validations:
+      required: false
+  - type: textarea
+    id: expected_behavior
+    attributes:
+      label: Expected behavior
+      description: Provide a clear and concise description of what you expected the SDK to do.
+    validations:
+      required: false
+  - type: input
+    id: affected_sdk_versions
+    attributes:
+      label: Affected SDK versions
+      description: What are the SDK versions you're seeing this bug in? 
+    validations:
+      required: true
+  - type: input
+    id: last_working_sdk_version
+    attributes:
+      label: Latest working SDK version
+      description: What was the last SDK version that was working as expected?
+    validations:
+      required: true
+  - type: dropdown
+    id: checked_lastest_sdk
+    attributes:
+      label: Did you confirm if the latest SDK version fixes the bug?
+      options:
+        - 'Yes'
+        - 'No'
+    validations:
+      required: true
+  - type: dropdown
+    id: integration_method
+    attributes:
+      label: Integration Methods
+      options:
+        - NPM
+        - Yarn
+        - Source
+    validations:
+        required: true
+  - type: input
+    id: rn_version
+    attributes:
+      label: React Native Version
+      description: e.g. 0.73
```

```suggestion
      description: e.g. 0.73.3
```
We already had bugs on patch versions of RN, so I think we should include the patch as well in the example, what do you think?